styles.css

```diff
@@ -0,0 +1,521 @@
+  display: none;
+}
+
+.header {
+  background-color: #fff;
+}
+
+#menu {
+  width: 25px;
+  height: 40px;
+  cursor: pointer;
+  margin-left: 20px;
+  margin-top: 10px;
+}
+
+.menuList {
+  margin: 0 20px;
+  transition: 0.3s ease;
+  display: none;
+}
+
+.menuList li {
+  list-style: none;
+  padding: 16px 0;
+}
+
+.menuList li a {
+  color: #272a31;
+  text-decoration: none;
+  font-family: 'Cocogoose Pro', sans-serif;
+}
+
+.contact-list > li > a {
+  color: #272a31;
+}
+
+.menuList li a:hover {
+  font-weight: bold;
+}
+
+.line {
+  height: 5px;
+  width: 100%;
+  background-color: #272a31;
+  display: block;
+  border-radius: 5px;
+  transition: 0.3s ease;
+}
+
+#line1 {
+  transform: translateY(-4px);
+}
+
+#line3 {
+  transform: translateY(4px);
+}
+
+.change .line {
+  background-color: #272a31;
+}
+
+.change #line1 {
+  transform: translateY(4px) rotateZ(-45deg);
+}
+
+.change #line3 {
+  transform: translateY(-6px) rotateZ(45deg);
+}
+
+.change #line2 {
+  opacity: 0;
+}
+
+.change {
+  display: block;
+}
+
+.menu-pop {
+  z-index: 1;
+  border-radius: 5%;
+  transition: 0.3s ease;
+}
+
+#menu-pop {
+  z-index: 2;
+}
+
+.menuList #menu-pop {
+  top: 0;
+  left: 0;
+  position: absolute;
+}
+
+.intro {
+  position: relative;
+  background-color: #d3d3d3;
+}
+
+.intro::before {
+  content: '';
+  position: absolute;
+  top: 0;
+  left: 0;
+  right: 0;
+  bottom: 0;
+  background-image: url(./img/background1.avif);
+  background-repeat: no-repeat;
+  background-size: cover;
+  opacity: 0.25;
+}
+
+.intro1 {
+  font-size: 13px;
+  width: 300px;
+  border: 1px solid #fff;
+  position: relative;
+  text-align: center;
+  margin-left: 40px;
+}
+
+.title3 {
+  position: relative;
+  font-family: 'Lato', sans-serif;
+  color: #ec5342;
+  font-size: 20px;
+  margin-bottom: 20px;
+  margin-left: 20px;
+}
+
+.title {
+  position: relative;
+  font-family: 'Cocogoose Pro', sans-serif;
+  color: #ec5342;
+  font-size: 30px;
+  margin-bottom: 20px;
+  text-align: center;
+}
+
+.intro-date {
+  position: relative;
+  font-family: 'Cocogoose Pro', sans-serif;
+  color: #272a31;
+  font-size: 20px;
+  margin-left: 40px;
+  margin-top: 20px;
+}
+
+.intro-location {
+  position: relative;
+  font-family: 'Lato', sans-serif;
+  color: #272a31;
+  font-size: 14px;
+  margin-left: 30px;
+  margin-top: 15px;
+}
+
+.program {
+  background-image: url(./img/background2.avif);
+  background-repeat: no-repeat;
+}
+
+.title1 {
+  text-align: center;
+  color: #d3d3d3;
+  font-family: 'Lato', sans-serif;
+  font-size: 20px;
+  margin-bottom: 5px;
+}
+
+.hr1 {
+  width: 40px;
+  margin: 0 auto;
+  color: #ec5342;
+}
+
+.list1 {
+  font-family: 'Cocogoose Pro', sans-serif;
+  text-decoration: none;
+  color: #ec5342;
+  font-size: 11px;
+}
+
+.info {
+  font-family: 'Lato', sans-serif;
+  color: #272a31;
+  font-size: 13px;
+  width: 200px;
+  margin-left: 100px;
+  margin-top: -20px;
+}
+
+.info-list li {
+  margin-left: 30px;
+  margin-top: 15px;
+  width: 300px;
```

Comment:
- [ ] Please make sure you add icons and give the `paragraph` the right color 🙏 

Original Mobile Design | Your Mobile Design(Home Page)
------------- | -------------
![image](https://user-images.githubusercontent.com/74814780/181597207-cb4acf5c-c865-4864-ade6-75eb0dbb6a25.png)  |  ![image](https://user-images.githubusercontent.com/74814780/181597249-eb68b9f7-8969-4c0d-832f-5472a44a879d.png)